ArcGISRuntimeSDKQt_CppSamples/Layers/ListKmlContents/ListKmlContents.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include <QDir>
 #include <algorithm>
+#include <QStandardPaths>
```

Comment:
I recommend keeping these alphabetical. The Qt team's git hooks will do this automatically if you have them installed.

This recommendation applies throughout.